internal/testing/backend/main_test.go

```diff
@@ -37,63 +37,30 @@ const (
 )
 
 var skipMatrix = map[string]map[string]bool{
-	// get Scorecard ID: ent: scorecard not found
-	"TestCertifyScorecard": {ent: true},
-	// input: IngestScorecards failed with err: get Scorecard ID: ent: scorecard not singular
-	"TestIngestScorecards": {ent: true},
-	// ent vuln query / novuln seems to be off
-	"TestIngestCertifyVulnerability": {ent: true},
-	// failed to execute IngestHasMetadata :: get HasMetadata: ent: has_metadata not singular
-	"TestHasMetadata": {ent: true},
-	// input: IngestBulkHasMetadata failed with element #1 {Type:pypi Namespace:<nil> Name:tensorflow Version:<nil> Qualifiers:[] Subpath:<nil>} with err: failed to execute IngestHasMetadata :: get HasMetadata: ent: has_metadata not singular
-	"TestIngestBulkHasMetadata": {ent: true},
-	// input: IngestHasSbom :: input: IngestHasSbom ::  ent: bill_of_materials not singular
 	// arango fails IncludedOccurrences_-_Valid_Included_ID and IncludedDependencies_-_Valid_Included_ID
-	"TestHasSBOM": {ent: true, arango: true},
-	// input: IngestHasSBOMs failed with err: input: IngestHasSbom :: input: IngestHasSbom ::  ent: bill_of_materials not singular
-	"TestIngestHasSBOMs": {ent: true},
-	// ent hash equal querying seems to be off
-	"TestHashEqual":        {ent: true},
-	"TestIngestHashEquals": {ent: true},
-	// ent is filling in "StartedOn" when not provided on input, but "FinishedOn" is.
-	"TestHasSLSA": {ent: true},
-	//  ent: source_name not singular
-	"TestHasSourceAt": {ent: true},
-	//  input: IngestHasSourceAts failed with err: ent: source_name not singular
-	"TestIngestHasSourceAts": {ent: true},
-	// ent: dep pkg querying subpath not working
+	"TestHasSBOM": {arango: true},
 	// keyvalue: failing on dep package querying
-	"TestIsDependency": {ent: true, memmap: true, redis: true, tikv: true},
+	"TestIsDependency": {memmap: true, redis: true, tikv: true},
 	// arango errors when ID is not found
-	// ent errors when ID is not found
-	"TestOccurrence": {ent: true, arango: true},
+	"TestOccurrence": {arango: true},
 	// ent: Path/Nodes/Neighbors not implemented
 	// keyvalue: path: input: No path found up to specified length
 	// neighbors: sorting not done, testdata is only in order for arango
 	"TestPath":      {ent: true, memmap: true, redis: true, tikv: true},
```

Comment:
`path`, `nodes` and `neighbors` are remaining as they have not been fully implemented yet